### PR TITLE
Updated about me section; worked on index.html and styling.css

### PR DESCRIPTION
**Pull request by Maaz4324**

### CODE REVIEW

**Comment by fykaa** [comment]
this is a minute edit, Maybe add a pop-up animation to it. 
You can improve UI for the Navbar by adding animation and smooth gestures to it and that of the tables section belw.

**Comment by Maaz4324** [comment]
Do you want navbar responsiveness fixed?

**Comment by Maaz4324** [comment]
Working on it.

File: index.html [hunk 1]
```diff
@@ -54,35 +57,39 @@ <h1>Portfolio</h1>
         </ul>
       </nav>
     </header>
+    <button>menu</button>
```
Comment (fykaa) [inline index.html:60]: remove this button

**Comment by fykaa** [comment]
lgtm , Resolve Conflicts

**Pull request closed without merging**